test/unit/rules/no-at-ember-render-modifiers-test.js

```diff
@@ -6,67 +6,133 @@ generateRuleTests({
 
   config: true,
 
-  good: ['<div {{this.someModifier}}></div>', '<div {{someModifier}}></div>'],
+  good: [
+    '<div {{this.someModifier}}></div>',
+    '<div {{someModifier}}></div>',
+    '<div {{did-foo}}></div>',
+    // helper -- a different rule should prevent this
+    // https://github.com/buschtoens/ember-render-helpers (depending on usage)
+    '{{did-insert}}',
+    '{{did-update}}',
+    '{{will-destroy}}',
+  ],
 
   bad: [
     {
       template: '<div {{did-insert}}></div>',
       verifyResults(results) {
-        expect(results).toMatchInlineSnapshot(`
-          [
-            {
-              "column": 5,
-              "endColumn": 19,
-              "endLine": 1,
-              "filePath": "layout.hbs",
-              "line": 1,
-              "message": "${MESSAGES['did-insert']}",
-              "rule": "no-at-ember-render-modifiers",
-              "severity": 2,
-              "source": "<div {{did-insert}}></div>",
-            },
-          ]
-        `);
+        expect({ results }).toMatchInlineSnapshot(
+          {
+            results: [
+              {
+                column: 5,
+                endColumn: 19,
+                endLine: 1,
+                filePath: 'layout.hbs',
+                line: 1,
+                message: MESSAGES['did-insert'],
+                rule: 'no-at-ember-render-modifiers',
+                severity: 2,
+                source: '<div {{did-insert}}></div>',
+              },
+            ],
+          },
+          `
```

Comment:
Looks like there's some old leftover snapshots in these tests. There should only be one arg to `toMatchInlineSnapshot`.

Reply:
the lint prevented from using a string, so I had to change to a real object.
then, when jest was running, it updated the string stuff after success -- the string part is the real snapshot persisted by jest, and the js object is the "expected" and helps out with the diff and generated the snapshot if the actual matches the expected.

Is this not how it's supposed to work?

Reply:
Jest's auto-updating of inline snapshots is buggy. It left the old snapshot in there. Just clear out the snapshots and let it regenerate without the extra arg.

Reply:
There are two arguments passed to `toMatchInlineSnapshot` in your code. There should only be one. One is a duplicate of the other. You need to delete the arguments to `toMatchInlineSnapshot` and re-run jest to regenerate the snapshots.

Reply:
but if I do that, jest adds the second argument back -- no diff is seen when doing `git status` / `git diff`.

To prove, I've added two commits:
- first removes the second argument, as you suggest
- second, is what happens after running the tests again

diff:
```
❯ git diff HEAD~2

ember-template-lint on  no-at-ember-render-modifiers [⇡] 
```

Reply:
thanks! tho, now I don't know why the lint rule I was seeing isn't triggering :sweat_smile: